src/configMeta.ts

```diff
@@ -24,6 +26,11 @@ export const enum ConfigVars {
    */
   LOCAL_WEB_SERVER_IDENTITY_TOKEN = 'local-web-server-identity-token',
 
+  /**
+   * The path to certificate and private key files
+   */
+  LOCAL_WEB_SERVER_HTTPS_CERT_AND_KEY_FILES = 'https',
```

Comment:
Can the value here be something less generic? E.g. `local-web-server-certificate-files`?

Reply:
Sure